pliers/extractors/api.py

```diff
@@ -138,31 +152,35 @@ def __init__(self, app_id=None, app_secret=None, model=None,
                                  "must be passed the first time a Clarifai "
                                  "extractor is initialized.")
 
-        self.tagger = ClarifaiApi(app_id=app_id, app_secret=app_secret)
-        if model is not None:
-            self.tagger.set_model(model)
-
-        self.model = model
-
-        if select_classes is None:
-            self.select_classes = None
-        else:
-            self.select_classes = ','.join(select_classes)
-
-        self._batch_size = self.tagger.get_info()['max_batch_size']
+        self.api = ClarifaiApp(app_id=app_id, app_secret=app_secret)
+        self.model = self.api.models.get(model)
+        self.min_value = min_value
+        self.max_concepts = max_concepts
+        self.select_concepts = select_concepts
+        if select_concepts:
```

Comment:
Should probably check to make sure this is a list (or just `listify` it), otherwise if the user passes a string it'll turn each character into a separate concept.